packages/cli/src/oclif/commands/team/remove.js

```diff
@@ -8,21 +8,31 @@ const {
 } = require('../../../utils/api');
 const { BASE_ENDPOINT } = require('../../../constants');
 
-const roleName = (role) => (role === 'collaborator' ? 'admin' : 'subscriber');
+const roleName = (role) =>
+  role === 'collaborator'
```

Comment:
if we're keeping this code around long term, it may be worth combining this into a function/mechanism that all of add/remove/get can use

Reply:
on board!